script/test-unit

```diff
@@ -1,26 +1,4 @@
 #!/bin/bash
 set -e
 
-source "$(dirname "$BASH_SOURCE")/.build"
-
-find_dirs() {
-  (
-    find . -not \( \
-        \( \
-          -path './vendor/*' \
-        \) -prune \
-      \) -name '*_test.go' -print0 | xargs -0n1 dirname | sed 's|^\./||' | sort -u
-  )
-}
-
-TEST_FLAGS=("${BUILD_FLAGS[@]}" -cover -coverprofile=cover.out)
-
-if [ -z "$TEST_DIRS" ]; then
-  TEST_DIRS=$(find_dirs '*_test.go')
-fi
-
-TESTS_FAILED=()
-
-for dir in $TEST_DIRS; do
-  go test "${TEST_FLAGS[@]}" "./${dir}"
-done
+go test -race -cover -v $( go list github.com/kubernetes-incubator/kompose/... | grep -v '/vendor/' )
```

Comment:
This looks much cleaner. To be honest I don't know why it was done using `find`. All other projects that I've looked at are also using `find`.
Lets try this and see what it will do ;-)

I would do just one change. To make sure same flags are used for tests and build. Can you add `BUILD_FLAGS` from `.build`?